frontend/src/app/patients/PatientForm.jsx

```diff
@@ -23,6 +23,18 @@ import { displayFullName, showError, showNotification } from "../utils";
 import "./EditPatient.scss";
 import Alert from "../commonComponents/Alert";
 
+const GET_FACILITIES_QUERY = gql`
```

Comment:
The facilites have already been queried by the app at this point I think we should use `useSelector()` to grab them. see: https://github.com/CDCgov/prime-simplereport/blob/main/frontend/src/app/commonComponents/Header.tsx

Reply:
Yeah I suppose we already have tech debt and a ticket to get rid of Redux so this doesn't make it a lot worse. Seems like the facility list is relatively static so as long as people don't leave the app open a long time it will be fresh enough.